frontend/src/components/involvement/get_involved.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import './get_involved.css';
+// import icon from 
+
+function GetInvolved() {
+  const arrayCards = [
+    [
+      'Report Issues',
+      'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi, magni.'
+    ], [
+      'Code Fixes', 'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Improve Documentation',
+      'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Propose Ideas', 'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Improve Documentation',
+      'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Improve Documentation',
+      'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ]
+  ]
+  return (
+    <main>
+      <div id="hero">
+        <div id="heading">
+          <h1 id="title">GET INVOLVED!</h1>
+        </div>
+      </div>
+      <div id="allCards">
+        {arrayCards.map(element => {
```

Comment:
provide key to the div.. put ( i ) as a argument in map function and just pass i as key to div